### PR TITLE
add generic function for arranger (use in clin)

### PR DESCRIPTION
**Pull request by francisl**
# feat

## Description

add arragner helper functions
plus generic sqon function need in clin (also verbatim in kf)

### CODE REVIEW

File: packages/ui/src/data/arranger/formatting.ts [hunk 1]
```diff
@@ -0,0 +1,28 @@
+export const MISSING_DATA = '__missing__';
```
Comment (evans-g-crsj) [inline packages/ui/src/data/arranger/formatting.ts:1]: No data could be in a const too.

File: packages/ui/src/data/arranger/formatting.ts [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+
```
Comment (evans-g-crsj) [inline packages/ui/src/data/arranger/formatting.ts:2]: capitalization: I see  (T)rue/(F)alse and (t)rue/(f)alse . Is that wanted?

File: packages/ui/Release.md [hunk 3]
```diff
@@ -1,3 +1,5 @@
+### 2.1.0 | 2021-10-08
```
Comment (oliviercperrier) [inline packages/ui/Release.md:1]: Pourquoi 2.1.0 ? Je viens de pousser 2.0.4

File: packages/ui/src/data/arranger/formatting.ts (hunk 2 above)
Comment (francisl) [reply packages/ui/src/data/arranger/formatting.ts:2]: this is arranger idiosyncrasy. I can move them in a dictionnary

**Review by oliviercperrier — APPROVED** [review]

File: packages/ui/src/data/arranger/formatting.ts (hunk 2 above)
Comment (francisl) [reply packages/ui/src/data/arranger/formatting.ts:7]: made a fix, using a dictionnary to regroup specific arrager keys and values

**Review by evans-g-crsj — APPROVED** [review]

**Pull request merged**